build.assets/Dockerfile-arm

```diff
@@ -82,3 +81,23 @@ RUN groupadd ci --gid="$GID" -o && \
     chown -R ci /var/lib/teleport
 
 VOLUME ["/go/src/github.com/gravitational/teleport"]
+
+# Install Rust.
```

Comment:
Instead of adding rust cannot we just match what we're doing in other builds and use node container to build WebUI as the wasm support is already there?

Reply:
Good call, closing in favor of that idea